prow/cmd/deck/csrf.md

```diff
@@ -0,0 +1,27 @@
+# CSRF attacks
+In Deck, we make a number of `POST` requests that require user authentication; namely, the request to `/pr-data.js` 
+to fetch a user's PR status, and the request to `/rerun` to trigger a job rerun. These requests are susceptible
+to [cross site request forgery (CSRF) attacks](https://en.wikipedia.org/wiki/Cross-site_request_forgery), 
+in which a malicious actor tricks an already authenticated user into submitting a form to one of these endpoints 
```

Comment:
It may be worth mentioning that CSRF can also be accomplished by tricking an authenticated user into making a GET request, which we do _not_ protect against. In order to be considered opted in to CSRF protection (as well as to be semantically correct), any request that mutates state must be a POST request.